.changeset/thirty-bikes-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vygruppen/spor-react": minor
```

Comment:
This is a major, aka a breaking change.

There are two ways you can deal with that:

You can deprecate the old names (but keep supporting them) and add the new ones. You could then add runtime logging that logs an error to the console if you use the old ones.

Alternatively, you can cut a breaking change.

Either way, you need to add a bit more info about what has changed, and what the users should do to migrate their code. If I used to have a card – what should I use instead? Etc.